pokedex/db/load.py

```diff
@@ -431,9 +431,13 @@ def dump(session, tables=[], directory=None, verbose=False, langs=None):
         # if specified, or for official languages by default.
         # For non-translation tables, dump all rows.
         if 'local_language_id' in columns:
-            if langs is None:
+            # If no lang arguments were passed or the 'all' argument was passed
+            if langs is None or langs == ['all']:
```

Comment:
I think `all` and `none` can be handled in `main.py`, similar to how `command_load` does it: https://github.com/veekun/pokedex/blob/e5c18c410994d5fb589bc3dceaba71f85268edfb/pokedex/main.py#L232-L235